File: referralhero/src/main/java/com/sdk/rh/AndroidIdUtil.java
package com.sdk.rh;

import android.content.Context;
import android.provider.Settings.Secure;

public class AndroidIdUtil {

    /**
     * <p>this method to return the ANDROID_ID value is unique to each user </p>
     *
     * @param context      pass Context.
     */
    public static String getAndroidId(final Context context) {
        return Secure.getString(context.getContentResolver(), Secure.ANDROID_ID);
    }
}

File: referralhero/src/main/java/com/sdk/referral/AndroidIdUtil.kt
package com.sdk.referral

import android.content.Context
import android.provider.Settings.Secure

object AndroidIdUtil {
    /**
     *
     * this method to return the ANDROID_ID value is unique to each user
     *
     * @param context      pass Context.
     */
    fun getAndroidId(context: Context): String {
        return Secure.getString(context.contentResolver, Secure.ANDROID_ID)
    }
}